docs/use_cases/recommender_systems.md

```diff
@@ -1,32 +1,31 @@
+<!--SEO SUMMARY: According to an IBM study, personalized product recommendations can lead to a 10-30% increase in revenue. However, creating truly personalized experiences requires extensive user and product data, which is not always available. For example, providers with restrictions on collecting user data may only have basic product information like genre and popularity. While collaborative filtering approaches can still work reasonably well, they leave significant gains on the table. Incorporating side data with a recommender based based on collaborative filtering can significantly increase recommendation quality (> 20% gains in precision), leading to greater customer satisfaction, more traffic, millions in additional revenue. -->
 
-# Recommender Systems
+# A Recommender System: Collaborative Filtering with Sparse Metadata
 
 ![](assets/use_cases/recommender_systems/recommender.jpg)
 
-Recommender Systems are becoming increasingly important, given the plethora of products offered to the users/customers. For example, in early days, fashion retailers developed basic versions of content-based recommenders that served quite well, but soon the need emerged for more relevant recommendations that could increase user’s engagement. At the moment of this writing, fashion retailers use far more sophisticated approaches that make use not only of the user’s purchasing/viewing history, but also of user and item metadata: user age, location, spending habits, mood, etc.; item category, popularity, etc.
+Recommender Systems are becoming increasingly important, given the plethora of products offered to users/customers. Some twenty years ago fashion retailers developed basic versions of content-based recommenders that did increase users engagement (compared with the variant which offered no recommendations), but as soon as the capabilities of the tracking systems improved, 
+it became possible to integrate new signals that could help provide even better recommendations. At the moment of this writing, fashion retailers use far more sophisticated approaches that use not only users' purchasing/viewing history, but also of user metadata (age, location, spending habits, mood, etc.) and item metadata (category, popularity, etc.)
```

Comment:
I think it's beside the point if the retailers use a CF approach. Some do, some don't. What exactly did you mean?

Reply:
Okay, now I see what you're doing with the paragraphs, which wasn't clear before. 
I would change line 10 below to more explicitly link these two cases:
instead of "However, it sometimes the case that only scarce side..."
say "But not everyone has access to the kind or amount of metadata fashion retailers do. Sometimes only scarce side...